js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationSensorRow.tsx

```diff
@@ -132,15 +141,38 @@ export const VirtualizedAutomationSensorRow = forwardRef(
           </RowCell>
           <RowCell>
             <Box flex={{direction: 'row', gap: 8, alignItems: 'flex-start'}}>
-              {/* Keyed so that a new switch is always rendered, otherwise it's reused and animates on/off */}
+              {/* Left aligned content */}
+              <Box flex={{grow: 1, gap: 8}}>
+                {/* Keyed so that a new switch is always rendered, otherwise it's reused and animates on/off */}
+                {sensorData ? (
+                  <SensorSwitch key={name} repoAddress={repoAddress} sensor={sensorData} />
+                ) : (
+                  <div style={{width: 30}} />
+                )}
+                <Link to={workspacePathFromAddress(repoAddress, `/sensors/${name}`)}>
+                  <MiddleTruncate text={name} />
+                </Link>
+              </Box>
+              {/* Right aligned content */}
               {sensorData ? (
-                <SensorSwitch key={name} repoAddress={repoAddress} sensor={sensorData} />
+                <Tooltip
+                  canShow={sensorData.sensorType !== SensorType.STANDARD}
+                  content="Testing not available for this sensor type"
+                  placement="top-end"
+                >
+                  <Button
+                    disabled={sensorData.sensorType !== SensorType.STANDARD}
+                    onClick={() => {
+                      setShowTestTickDialog(true);
+                    }}
+                    style={{height: '24px', marginTop: '-4px'}} // center button text with content in AutomationRowGrid
+                  >
+                    Manual tick
```

Comment:
I think others have probably weighed in already, but I have a slight preference for "Evaluate tick" instead of "Manual tick" because evaluate is a clearer verb and it'd help clarify that this is a button and not a tag / outlined text label.